src/content/docs/logs/logs-context/disable-automatic-logging.mdx

```diff
@@ -123,7 +123,7 @@ After you disable via the New Relic UI, it may take several minutes for the agen
     Client-side configuration settings work completely independently of server-side configuration. The config setting to disable auto logging takes precedence even if the server-side configuration takes precedence for the agent (like Ruby) in other situations. Only the agents that read the changed config file will be impacted.
     
     <Callout variant="important">
-    Additional logging metrics are collected and displayed in the Logs chart on the **APM Summary** page. If you disable log forwarding through the Account Settings UI, these logging metrics will not be affected. However, if you need to disable the metrics, you'll need to explicitly disable them in your application's APM configuration.
+    Additional logging metrics are displayed in the **Logs** chart on the APM **Summary** page. If you disable log forwarding, these logging metrics won't be affected. To disable these metrics, you must do this via APM agent configuration (for example, [this config option for PHP](/docs/apm/agents/php-agent/configuration/php-agent-configuration/#cfg-application_logging_metrics-enabled)).
```

Comment:
@zuluecho9 I was surprised to see PHP singled out in this callout when there's a list of all the other agents that follows. Do we need this PHP link in this callout? Isn't it sufficient to tell users to check out the various links that we provide?